dashboard/test/ui/features/code_tools/google_blockly/modal_function_editor.feature

```diff
@@ -31,7 +31,7 @@ Scenario: Can edit a function
   # Open Sprites flyout
   And I press "blockly-d"
   # Drag block to top of function
-  And I drag block "new-sprite-block" to offset "40, 100"
+  And I drag block "new-sprite-block" to offset "40, 105"
```

Comment:
I think you'll need to make the same update [here](https://github.com/code-dot-org/code-dot-org/blob/3c512ad40adb5e38a02e9918254efa3c5358af1b/dashboard/test/ui/features/code_tools/google_blockly/modal_function_editor_eyes.feature#L21). It would be a good idea to run that test and [this one](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/ui/features/code_tools/google_blockly/spritelab_eyes.feature) locally to check that the eyes tests are still working as expected.

Reply:
Oh good call out. Do you think I didn't run into this yet because drone doesn't run eyes tests?

Reply:
yep!